.github/workflows/ci.yml

```diff
@@ -4,6 +4,9 @@
 name: ci
 on:
   pull_request:
+    paths-ignore:
+      - '*.md'
+      - 'VERSION'
```

Comment:
I guess this will be good for speed up the release process, but if the VERSION file contains a non-semver string inside then tests fail.

Reply:
That issue can be solved by adding a `make version/check` that validates the semver string in `VERSION`.
However, I found another issue: when we bump the semver, it always causes the testdata to change, and then full ci checks in GHA would kick in, resulting in no speed up in the release process.

Do you know why the test data is tied to the `VERSION` string? 🤔 Would it be possible to not depend on `VERSION` to generate test data?

Reply:
```
generate_file "cloud.data.json" {
  lets {
    version = tm_trimspace(tm_file("${terramate.root.path.fs.absolute}/VERSION"))
    well_known = {
      required_version = "= ${tm_replace(let.version, "/-.*/", "")}"
    }
  }
  content = tm_jsonencode({
    orgs       = global.testserver.orgs,
    users      = global.testserver.users,
    well_known = let.well_known,
  })
}
```

We use this to test the `/.well-known/cli.json` `required_version` field. So it uses the current version (without `-.*`) as basis.

Reply:
@waxb please change this to be:

```
   lets {
    version = "> 0.4.3"
    ...
   }
```

it should work for all newer releases, then there's no test dependency on `VERSION` anymore and we unblock this.

Reply:
Makes sense, fixed it here: https://github.com/terramate-io/terramate/pull/1423